dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/report/TrackerErrorCode.java

```diff
@@ -117,6 +117,7 @@
     E1122( "Missing required enrollment property: `{0}`." ),
     E1123( "Missing required event property: `{0}`." ),
     E1124( "Missing required relationship property: `{0}`." ),
+    E1125( "Value {0} is not a valid option for {1} in option set {2}" ),
```

Comment:
One suggestion here would be to specify the type of the object (Attribute, DataElement) because for the client can be difficult to understand where is the error just from the uid.
Maybe something like this:
1125( "Value {0} is not a valid option for {1}({2}) in option set {3}" )
Where 2 is either "Attribute" or "DataElement"